feature_extraction/COVAREP_feature_formant_extraction_perfile.m

```diff
@@ -291,6 +307,9 @@
     results = [results array2table(signal, 'VariableNames', names)];
 end
 
+%% adjust time to account for different start point
+results.time = results.time + options.start;
```

Comment:
options.start has to be updated according to start_index before added to results.time.
options.start = (start_index-1)/fs % I think

Reply:
good point! `options.start` might not be the same as `(start_index-1)/fs` if there have been rounding issues. I will push a change in a few minutes

Reply:
Exact. Great thanks!